mteb/abstasks/AbsTaskClusteringFast.py

```diff
@@ -138,9 +138,15 @@ def _evaluate_subset(
             max_depth=self.max_depth,
             rng_state=rng_state,
         )
-        all_v_scores = itertools.chain.from_iterable(v_measures.values())
-        mean_v_measure = np.mean(list(all_v_scores))
-        scores = {"v_measures": v_measures, "v_measure": float(mean_v_measure)}
+        v_measures = list(itertools.chain.from_iterable(all_v_scores.values()))
+
+        mean_v_measure = np.mean(v_measures)
+        v_std = np.std(v_measures)
+        scores = {
+            "v_measures": v_measures,
```

Comment:
Culprit for https://github.com/embeddings-benchmark/mteb/pull/892/files#r1632039634
```suggestion
            "v_measures": all_v_scores,
```

Reply:
Ah right so we just want to make it such that all have the "Level" regardless of whether it is hierarchical or not.